bigchaindb/core.py

```diff
@@ -77,6 +130,9 @@ def check_tx(self, raw_transaction):
             raw_tx: a raw string (in bytes) transaction.
         """
 
+        chain = self.bigchaindb.get_latest_abci_chain()
+        self.abort_if_abci_chain_is_not_synced(chain)
```

Comment:
As discussed in our office conversation it would be better to load the genesis for the fork during `__init__()` and then use it inside `info`, `init_chain` and `check_tx`